From 480d9da302885d9e8b3d1f9c68def569e3c8a8c0 Mon Sep 17 00:00:00 2001
From: Sergey Matyukevich <geomatsi@gmail.com>
Date: Mon, 9 May 2022 00:17:21 +0300
Subject: [PATCH] Makefile: remove default endianness

There is no way to remove configuration options from EXTRA_CFLAGS
without editing Makefile. So default LE configuration can not be
conveniently replaced by BE. Remove default endianness from the
Makefile to enable its configuration via USER_EXTRA_CFLAGS.

Signed-off-by: Sergey Matyukevich <geomatsi@gmail.com>
---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index dfca305..fde4e6b 100644
--- a/Makefile
+++ b/Makefile
@@ -22,7 +22,6 @@ endif
 
 EXTRA_CFLAGS += -I$(src)/include
 
-EXTRA_CFLAGS += -DCONFIG_LITTLE_ENDIAN
 EXTRA_CFLAGS += -DCONFIG_IOCTL_CFG80211 -DRTW_USE_CFG80211_STA_EVENT
 
 EXTRA_LDFLAGS += --strip-debug
-- 
2.35.1

